apps/examples/testcase/le_tc/kernel/tc_timer.c

```diff
@@ -305,7 +309,8 @@ static void tc_timer_timer_initialize(void)
 	}
 
 	/* check the count for g_alloctimers and g_freetimers after timer_initialize now they change to original value */
-	timer_initialize();
+	ret = ioctl(fd, TESTIOC_TIMER_INITIALIZE, NULL);
+	TC_ASSERT_EQ("timer_initiallize", ret, OK);
```

Comment:
This return check is not needed.
```
(void)ioctl(XXX);
```
is enough.

Reply:
okay. good. thanks